LICENSE-APACHE

```diff
@@ -187,7 +187,7 @@ Copyright (C) 2023  ZKID Labs AG
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright (C) 2023  ZKID Labs AG
+   Copyright (C) 2023  Zero Block Global Foundation
```

Comment:
According to 4.(C) of that Apache license you should include all authors into the license copyright.